contrib/email-notifications/auter-notify.module

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+#
+# Send an email notification with the status of the auter prepare stage.
+#
+# auter is a yum-cron type package which implements automatic updates on an
+# individual server with features such as predownloading packages and reboots.
+#
+#
+# Copyright 2016 Rackspace, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+function main() {
+  function logit() {
+    # If running on a tty, or the --stdout option is provided, print to screen
+    ( tty -s || [[ -n "$STDOUT" ]] ) && echo "$1"
+    logger -p INFO -t auter "$1"
+  }
+
+  # Setup
+  local STAGE="$1"
+  if [[ -f "${SCRIPTDIR}/auter-notify.conf" ]]; then
+    source "${SCRIPTDIR}/auter-notify.conf"
+  else
+    logit "ERROR: auter-notify.conf is missing, skipping $0"
+  fi
+
+  local MAIL_CC
+  MAIL_CC="-c $(sed 's/ / -c /g' <<< "$MAIL_CC")"
```

Comment:
If no CC address is provided then the variable will simply contain "-c " which will cause the mailx command on line 104 to fail